contrib/src/main/java/gov/nasa/jpl/aerie/contrib/serialization/rulesets/BasicValueMappers.java

```diff
@@ -23,14 +23,19 @@
 import gov.nasa.jpl.aerie.contrib.serialization.mappers.PrimitiveShortArrayValueMapper;
 import gov.nasa.jpl.aerie.contrib.serialization.mappers.ShortValueMapper;
 import gov.nasa.jpl.aerie.contrib.serialization.mappers.StringValueMapper;
+import gov.nasa.jpl.aerie.contrib.serialization.mappers.UnitValueMapper;
 import gov.nasa.jpl.aerie.merlin.framework.ValueMapper;
 import gov.nasa.jpl.aerie.merlin.protocol.types.Duration;
+import gov.nasa.jpl.aerie.merlin.protocol.types.Unit;
 
 import java.nio.file.Path;
 import java.util.List;
 import java.util.Map;
 
 public final class BasicValueMappers {
+
+  public static ValueMapper<Unit> $unit() { return new UnitValueMapper(); }
```

Comment:
I would highly recommend retaining a comment on this method explaining how its declaration order relative to `$enum()` is critically important. Declaration order in Java is not *usually* semantically relevant, but it definitely is here.

Reply:
Sure I can add the comment back. Sorry, I thought that was more for me to understand better. :P

Reply:
Why not both? ^_^